wire-ios/Wire-iOS Tests/Mocks/UserSessionMock.swift

```diff
@@ -55,15 +62,16 @@ final class UserSessionMock: UserSession {
     }
 
     func enqueue(_ changes: @escaping () -> Void) {
-        fatalError("not implemented")
+        changes()
     }
 
     func enqueue(_ changes: @escaping () -> Void, completionHandler: (() -> Void)?) {
         fatalError("not implemented")
     }
 
-    func addUserObserver(_ observer: ZMUserObserver, for: UserType) -> NSObjectProtocol? {
-        fatalError("not implemented")
+    func addUserObserver(_ observer: ZMUserObserver, for user: UserType) -> NSObjectProtocol? {
+        // temporary
+        return observer
```

Comment:
What do you mean by temporary? You could also return nil and it would work. Or you could return `NSObject()`. Usually the observer is storing the token so in this case the observer would have a reference to itself.

Reply:
Fixed it.